hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_websocket_adaptor.py

```diff
@@ -22,6 +24,27 @@ class BybitPerpetualWebSocketAdaptor:
     def __init__(self, websocket: aiohttp.ClientWebSocketResponse):
         self._websocket = websocket
 
+    @classmethod
+    def endpoint_from_raw_message(cls, raw_message: str) -> str:
+        message = ujson.loads(raw_message)
+        return cls.endpoint_from_message(message=message)
+
+    @classmethod
+    def endpoint_from_message(cls, message: Dict[str, Any]) -> str:
+        if message[cls._operation_field_name] is cls._subscription_operation:
+            return message[cls._payload_field_name][0]
+        else:
+            return message[cls._operation_field_name]
+
+    @classmethod
+    def payload_from_raw_message(cls, raw_message: str) -> Dict[str, Any]:
+        message = ujson.loads(raw_message)
+        return message
+
+    @classmethod
+    def payload_from_message(cls, message: str) -> Dict[str, Any]:
```

Comment:
It seems here like the payload is the whole message. That does not make much sense. Is that correct?